apollo-router/tests/integration/mod.rs

```diff
@@ -7,6 +7,8 @@ mod docs;
 mod file_upload;
 mod lifecycle;
 mod operation_limits;
+
+#[cfg(any(not(feature = "ci"), all(target_arch = "x86_64", target_os = "linux")))]
```

Comment:
```suggestion
```
part of the larger issue here is that nobody on the team is testing the router with a local redis instance. This has to change

Reply:
This change does exactly that. The previous logic disabled for platforms, but now it's only disabled for CI on specific platforms.

Developers running locally will run all the tests:

#[cfg(**any(not(feature = "ci"),** all(target_arch = "x86_64", target_os = "linux")))]

Reply:
ah, got it, thanks!